tests/backends/tests.py

```diff
@@ -32,6 +32,31 @@
 from . import models
 
 
+class DatabaseBackendTest(SimpleTestCase):
+
+    def test_initialization_class_attributes(self):
+        """
+        Check that the "initialization" class attributes like client_class
```

Comment:
State the expected behavior. No "Check that" or "Test that" prefixes since all tests do that.